atc/builds/tracker_test.go

```diff
@@ -1,123 +1,107 @@
 package builds_test
 
 import (
+	"context"
+	"testing"
 	"time"
 
-	"code.cloudfoundry.org/lager/lagertest"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
+	"code.cloudfoundry.org/lager"
 	"github.com/concourse/concourse/atc/builds"
 	"github.com/concourse/concourse/atc/db"
 	"github.com/concourse/concourse/atc/db/dbfakes"
 	"github.com/concourse/concourse/atc/engine"
 	"github.com/concourse/concourse/atc/engine/enginefakes"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 )
 
-var _ = Describe("Tracker", func() {
-	var (
-		fakeBuildFactory *dbfakes.FakeBuildFactory
-		fakeEngine       *enginefakes.FakeEngine
-
-		tracker *builds.Tracker
-		logger  *lagertest.TestLogger
-	)
+type TrackerSuite struct {
```

Comment:
Should this be part of this PR? Isn't this part of the other refactor?

Reply:
I put it here so that at the end the `atc/builds/` package can be fully one test framework, which is accomplished by the removal of the tracker runner, which is part of this PR.